README.md

```diff
@@ -8,10 +11,20 @@ Running Salsa inside of an iOS simulator will output two things into a specified
 Certain macOS-only APIs need to be used to encode text for .sketch files. Having two steps allows us to define our own intermediate file format that’s easier to work with than the full sketch file format. This means we can leverage this file format in the future if we want to expand this tool for other platforms.
 
 # Installing Salsa
+### Cocoapods
 ```ruby
 pod 'Salsa'
 ```
 
+### Carthage
+```ruby
+github 'Yelp/salsa'
+```
+```
+carthage update
+```
```

Comment:
nit: We shouldn't need to put the update line here, it should just be the Cartfile line needed like we have with CocoaPods. This is especially true since there are different ways users might be invoking Carthage.